pyproject.toml

```diff
@@ -4,7 +4,13 @@ version = "0.0.0"
 description = "a framework for creating intelligent agents within Microsoft Teams"
 readme = "README.md"
 requires-python = ">=3.11,<3.13"
-dependencies = ["agent-runtime", "teams_memory", "litellm", "evals"]
+dependencies = [
+    "agent-runtime",
+    "teams_memory",
+    "litellm",
+    "evals",
+    "pydantic>=2.0"
```

Comment:
NIT: why is this required at the root level?

Reply:
removed